test/new-e2e/tests/installer/linux_test.go

```diff
@@ -79,6 +79,14 @@ func TestDebianX86(t *testing.T) {
 	runTest(t, "dpkg", os.AMD64Arch, os.DebianDefault, true)
 }
 
+func TestSuseX86(t *testing.T) {
+	runTest(t, "rpm", os.AMD64Arch, os.SuseDefault, false)
```

Comment:
nice!
Unfortunately you should disable those tests, as all tests are now [disabled temporarily due to a breaking on going changes: ](https://github.com/DataDog/datadog-agent/blob/main/test/new-e2e/tests/installer/linux_test.go#L58-L80).
They'll be enabled back in the next day, and I'll remove the skip from Suse when it's time